Makefile

```diff
@@ -319,7 +319,7 @@ proto-swagger-gen:
 	@./scripts/protoc-swagger-gen.sh
 
 proto-lint:
-	@buf check lint --error-format=json
```

Comment:
So we don't have a 1.0.0 yet to check breaking against ... but we will soonish ...